docs/UserGuide.md

```diff
@@ -134,35 +184,51 @@ Format: `edit INDEX [c/COMPANY] [r/ROLE] [s/STAGE] [d/DATETIME] [e/COMPANY_EMAIL
 * You can remove all the internship’s tags by typing `t/` without
     specifying any tags after it.
 
+<div markdown="block" class="alert alert-success">
+**:bulb: Tip:** You can edit any number of attributes with the edit command!
+</div>
+
 Examples:
 * `edit 1 s/Behavioural Interview e/hr@meta.com` Edits the stage and email address of the first internship to be `Behavioural Interview` and `hr@meta.com` respectively.
 * `edit 2 l/golang t/` Adds `golang` as the only language tag for the second internship and clears all existing tags.
 
-### 4.3. Listing all existing internship applications: 
+### 6.3. Listing all existing internship applications: 
 
 Lists all of your internship applications in WorkBook.
```

Comment:
not sure if i recall correctly, are we moving the sorted stuff from Add into here?